Source/WebCore/Modules/reporting/ReportingObserver.idl

```diff
@@ -38,6 +39,6 @@ typedef sequence<Report> ReportList;
 };
 
 dictionary ReportingObserverOptions {
-    sequence<DOMString>? types;
+    sequence<[AtomString] DOMString>? types;
```

Comment:
Sigh... this should really have been an argument of observe, not the constructor.

Reply:
:-(